ansible/roles/rt/tasks/main.yml

```diff
@@ -1,38 +1,147 @@
-- name: check if our postgres container is there
-  shell: docker ps | tail -n +2 | awk '{ print $NF }' | sed 's/,/\n/g' | grep '^rt_pg$'
-  register: rt_pg
-  ignore_errors: True
-
-- name: docker postgres volumes
-  command: mkdir -p /data/rt_pg
-  when: rt_pg.rc == 1
-
-- name: docker postgres server
-  command: docker run -d -e="POSTGRES_DBA_PW={{postgres_dba_pw}}" -v="/data/rt_pg:/data" -p=5432 -name=rt_pg okfn/postgres:0.4
-  when: rt_pg.rc == 1
-
-#  - name: address of postgresql server
-#    set_fact: db_host={{docker_containers[0].NetworkSettings.IPAddress}}
-#
-#  - name: show db_host
-#    debug: msg="{{db_host}}"
-
-- name: check if our rt container is there
-  shell: docker ps | tail -n +2 | awk '{ print $NF }' | sed 's/,/\n/g' | grep '^rt$'
-  register: rt
-  ignore_errors: True
-
-- name: docker rt volumes
-  command: mkdir -p /data/{{ docker_container }}
-  when: rt.rc == 1
-
-- name: create docker rt server
-  command: docker run -d -e="RT_DATABASE_PW={{rt_database_pw}}" -v="/data/rt:/data" -p=127.0.0.1:5025:25 -p=127.0.0.1:5080:80 -link=rt_pg:db -name={{ docker_container }} okfn/rt:0.16
-  when: rt.rc == 1
-
-- name: configure docker rt server
-  copy: src=RT_SiteConfig.pm dest=/data/rt/rt_conf/ owner=root group=www-data mode=0640
-  notify: restart rt
+
+- name: Install RT pre-reqs through apt-get
+  apt: pkg={{ item }} state=installed
+  with_items:
+  - build-essential 
+  - postgresql-client 
+  - apache2 
+  - mailutils 
+  - openssl 
+  - libyaml-perl 
+  - libyaml-appconfig-perl 
+  - libssl-dev 
+  - libexpat1-dev
+  - libdbd-mysql-perl
+  - libnet-smtp-ssl-perl 
+  - libdatetime-event-ical-perl
+  - procmail
+
+- name: get latest RT release
+  get_url: url=https://download.bestpractical.com/pub/rt/release/rt-4.2.9.tar.gz dest=/opt/
```

Comment:
Can you make the version number a variable? Makes it easier to upgrade in the future.